File: app/src/main/java/com/mobile/model/SeatingsInfo.java
package com.mobile.model;

import org.parceler.Parcel;

import java.util.List;

/**
 * Created by anubis on 6/27/17.
 */

@Parcel
public class SeatingsInfo {

    public int rows;
    public int columns;
    public List<SeatInfo> seats;
}

File: app/src/main/java/com/mobile/model/SeatingsInfo.kt
package com.mobile.model

/**
 * Created by anubis on 6/27/17.
 */

class SeatingsInfo(val rows: Int = 0, val columns: Int = 0, val seats: List<SeatInfo>? = null) {
    val hasNoSeats: Boolean
        get() {
            return rows == 0 || columns == 0 || seats?.size ?: 0 == 0
        }

}
